### ISSUE TITLE
Intermittent logging out/page refresh after successfully logging in

### ISSUE THREAD

**Issue by acornestean** [mozilla-extensions/regrets-reporter#111]
Description:
After successfully logging in on either a regular or admin account, I keep getting logged out/the page refreshes (as far as I can tell).
This occurs about 3-4 times before I managed to log in and stay logged in.

Steps to reproduce:
1. Access http://94.3.209.235:8502/
2. Log in with either a regular or admin account
3. Notice the user is successfully logged in
4. Notice that almost immediately after successfully logging in, the page refreshes/the app logs out the user

Actual:
After successfully logging in, the page refreshes or the app logs me out. This occurs several times before I remain logged in.

Expected:
The log in process should function as expected on the first attempt.

https://user-images.githubusercontent.com/50236075/163793096-21dd53f2-3a7e-451a-871c-4773d964641a.mp4

**Comment by iRanadheer** [comment]
Hey @acornestean,

That's strange. It shouldn't happen like that. Thanks for bringing it to us. I've checked the app now, It's not logging me out. Can you try again whenever you get time?

**Comment by acornestean** [comment]
Hey @iRanadheer 

No longer occurs when logging in on regular accounts – from about 25 attempts, no logging out occurred. 

However, admin accounts still appear to be affected – from 30 attempts, the issue occurred more than half the time.

However, I think that the issue on admin accounts might be caused by the “save login” panel that appears from time to time, since the login attempt and the display of the panel almost coincide. 
Once the panel appears, the issue might still occur with a few future login attempts and then at some point it keeps me logged in.
If I select to remember the login details on the panel once it first appears, the issue still occurs but less frequent as far as I can tell?

So it might be the panel that causes this issue, but I’m not sure.

On regular account logins, the “save login” panel does not appear for some reason and I’m able to log in and stay logged in 100% of the time.

**Comment by iRanadheer** [comment]
Hey @acornestean,

when you say 25-30 attempts, do you mean after labeling 25-30 data points? In any case,  this shouldn't matter and so as the save login panel. We didn't set any timer to log out the user.  We will check the log statements if we see any error prints on the console. Can we close this issue? we can remain open too.

**Comment by acornestean** [comment]
Hi @iRanadheer ,

I now notice the wording I used might not have been very clear. I apologize !

By “attempt” I meant me trying to log in using an account with the end result of actually remaining logged in and not getting logged out due to the bug. It has nothing to do with labeling data points.

In the case of using a regular account, I never got logged out by the app during today’s testing. So the issue is presumably fixed for these accounts.

In the case of using an admin account, I managed to log in and stay logged in maybe 1 in 3-4 tries. The rest of the logging in’s ended up with me getting logged out almost immediately after logging in i.e. the bug occurred.

Hopefully it’s more clear this time, but if there’s anything I can detail more or is unclear, let me know.
